test/http-test.js

```diff
@@ -83,6 +83,23 @@ describe('HTTP', function() {
     await wallet.open();
   });
 
+  it('should enable seed phrase recovery', async () => {
+    const options = {
+      passphrase: 'PASSPHRASE'
+    };
+
+    const testwallet = await wclient.createWallet('testwallet', options);
+    assert.strictEqual(testwallet.master.encrypted, false);
```

Comment:
This is a good assertion here, but I think for consistency you should ALSO copy the `getMaster` test you use two more times below. You can call the variables `master1`, `master2` and `master3` for example.